src/debugger/adapter/DebugAdapter.js

```diff
@@ -126,9 +128,17 @@ class PrepackDebugSession extends DebugSession {
         process.exit();
       },
     };
+
+    this._ensureAdapterChannelCreated("launchRequest");
+    invariant(this._adapterChannel !== undefined, "Adapter Channel used before it was created, in debugger.");
```

Comment:
The same unnecessary.

Reply:
Same as above (line 66), flow.

Reply:
@yinghuitan since I need the invariant for flow, should I just keep the `  this._ensureAdapterChannelCreated` call for readability purposes?